mr/src/main/java/org/apache/iceberg/mr/hive/serde/objectinspector/IcebergObjectInspector.java

```diff
@@ -65,17 +65,26 @@
           .invoke();
 
   public static ObjectInspector create(@Nullable Schema schema) {
+    return create(schema, true);
+  }
+
+  public static ObjectInspector create(@Nullable Schema schema, boolean caseSensitive) {
```

Comment:
Why accept a case sensitivity setting for something that is only used by Hive? I thought that the purpose of this PR was to allow case sensitive behavior for the InputFormat, but not Hive because Hive's behavior is case insensitive.

Reply:
Thanks, comment addressed.